scratch/shed.h

```diff
@@ -39,6 +39,8 @@ typedef unsigned uint32_t;
 typedef void* DIR; // yikes
 typedef void* FILE; // yikes
 
+typedef void* char32_t; // yikes
```

Comment:
Wow, this is a struct? I agree, yikes.

Reply:
Just so you know, all these "yikes"-marked typedefs are not their actual types, they are just type names that I didn't bother looking up the definition for.

Though I wonder whether I am tricking reopt into making mistakes by providing incorrect types.